tools/mlir-opt/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ set(LIBS
   MLIRLLVMIR
   MLIRLoopOps
   MLIRNVVMIR
+  MLIROpenMPIR
```

Comment:
Do we need the IR suffix? MLIROpenMPIR -> MLIROpenMP? (since OpenMP is just a programming model, while NVVM, LLVM are IRs)